Game/MyProject/IvGraphics/OGL/IvFrameBufferOGL.cpp

```diff
@@ -0,0 +1,114 @@
+//
+//  IvFrameBufferOGL.cpp
+//  IvGraphics
+//
+//  Created by Monica Olanescu - (p) on 4/4/19.
+//
+
+#include "IvFrameBufferOGL.h"
+
+IvFrameBufferOGL::IvFrameBufferOGL():fbo(0)
+{
+    
+}
+
+IvFrameBufferOGL::IvFrameBufferOGL(const std::vector<IvRenderTarget*>& renderTargets)
+{
+    glGenFramebuffers(1,&fbo);
+    this->renderTargets=renderTargets;
+}
+
+IvFrameBufferOGL::~IvFrameBufferOGL()
+{
+    glDeleteFramebuffers(1,&fbo);
+}
+
+void IvFrameBufferOGL::Bind(int width,int height)
```

Comment:
This "Bind" function is more like a Create function that should be called once. An typical bind function for framebuffer looks something like this:
```suggestion
void IvFrameBufferOGL::Bind()
{
	if(this->m_created == false)
	{
		this->m_created = true;
		this->Create(this->m_width,this->m_height);
	}
	glBindFramebuffer(GL_FRAMEBUFFER,fbo);
}
```
where:
bool m_created = false;
uint32_t m_width = 1024; //default values
uint32_t m_width = 768;